app/controllers/bz_controller.rb

```diff
@@ -13,4 +13,30 @@ def last_user_url
 
     render :nothing => true
   end
+
+  def video_link
+    obj = {}
+    # For PoC I am using a manually generated list of links
```

Comment:
On Tue, Oct 27, 2015 at 06:03:36AM -0700, Brian Sadler wrote:
> If it's still true that you plan to create these links in an automated fashion, I'll merge this.  But if you think we'll end up using manually generated links -- let's at least read these from a config file.  Something not in source control.

It'll be in the database once the approach proves itself.